### PR TITLE
Feature/player controls

### PR DESCRIPTION
**Pull request by FilipGrebowski**
Player controls added to Player.

### CODE REVIEW

File: src/components/Core/Player/Player.jsx [hunk 1]
```diff
@@ -20,6 +24,12 @@ class Player extends React.Component {
     this.onScrub = this.onScrub.bind(this);
     this.handlePlay = this.handlePlay.bind(this);
     this.handlePause = this.handlePause.bind(this);
+
+    this.handleAddToLibrary = this.handleAddToLibrary.bind(this);
+    this.handleRepeat = this.handleRepeat.bind(this);
+    this.handleShuffle = this.handleShuffle.bind(this);
+    this.onVolumeChange = this.onVolumeChange.bind(this);
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.jsx]: `handleVolumeChange`

File: src/components/Core/Player/Player.jsx [hunk 2]
```diff
@@ -53,7 +63,80 @@ class Player extends React.Component {
   }
 
   handleNext() {
-    this.props.mk.instance.player.skipToNextItem();
+    const song = this.props.mk.instance.player
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.jsx]: This is a `player`, not a `song`

File: src/components/Core/Player/Player.jsx [hunk 3]
```diff
@@ -53,7 +63,80 @@ class Player extends React.Component {
...
+    song.skipToNextItem();
+
+    if (song.repeatMode === 1) {
+      song.seekToTime(0);
+    }
+  }
+
+  handleAddToLibrary() {
+    const music = this.props.mk.instance;
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.jsx]: `music` is redundant

File: src/components/Core/Player/Player.jsx [hunk 4]
```diff
@@ -53,7 +63,80 @@ class Player extends React.Component {
...
+    music.player.addToLibrary = 1;
+  }
+
+  handleRepeat() {
+    const music = this.props.mk.instance;
+    const {isRepeatOn} = this.state;
+
+    if (music.player.repeatMode === 0) {
+      this.setState({
+        isRepeatOn: true,
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.jsx]: State is useless in this case, it is already stored in `music.player`

File: src/components/Core/Player/Player.jsx [hunk 5]
```diff
@@ -53,7 +63,80 @@ class Player extends React.Component {
...
+      music.player.repeatMode = 0;
+    }
+  }
+
+  handleShuffle() {
+    const music = this.props.mk.instance;
+
+    if (this.state.isShuffleOn === false) {
+      this.setState({
+        isShuffleOn: true,
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.jsx]: State is useless in this case, it is already stored in music.player

File: src/components/Core/Player/Player.jsx [hunk 6]
```diff
@@ -141,21 +215,58 @@ class Player extends React.Component {
...
         <div className={styles.buttons}>
           <span onClick={this.handlePrevious}>
-            <i className='fas fa-backward'/>
+            <i className={'fas fa-backward'}/>
           </span>
           {mk.instance.player.isPlaying ? (
             <span className={styles.main} onClick={this.handlePause}>
-              <i className='fas fa-pause'/>
+
+              <i className={"fas fa-pause"}/>
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.jsx]: `'`

File: src/components/Core/Player/Player.jsx [hunk 7]
```diff
@@ -141,21 +215,58 @@ class Player extends React.Component {
...
             </span>
           ) : (
             <span className={styles.main} onClick={this.handlePlay}>
-              <i className='fas fa-play'/>
+              <i className={"fas fa-play"}/>
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.jsx]: `'`

File: src/components/Core/Player/Player.jsx [hunk 8]
```diff
@@ -141,21 +215,58 @@ class Player extends React.Component {
...
             </span>
           )}
           <span onClick={this.handleNext}>
-            <i className='fas fa-forward'/>
+            <i className={"fas fa-forward"}/>
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.jsx]: `'`

File: src/components/Core/Player/Player.jsx [hunk 9]
```diff
@@ -141,21 +215,58 @@ class Player extends React.Component {
...
           </span>
         </div>
+
+        <div className={styles.buttons}>
+
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.jsx]: no `\n`

File: src/components/Core/Player/Player.jsx [hunk 10]
```diff
@@ -141,21 +215,58 @@ class Player extends React.Component {
...
+                  value={this.props.mk.instance.player.volume}
+                  onChange={this.onVolumeChange}
+                  min={0}
+                  max={1}
+                  step={0.01}
+                />
+              </div>
+            </div>
+          </span>
+
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.jsx]: no `\n`

File: src/components/Core/Player/Player.jsx [hunk 11]
```diff
@@ -141,21 +215,58 @@ class Player extends React.Component {
...
+        </div>
+
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.jsx]: no `\n`

File: src/components/Core/Player/Player.scss [hunk 12]
```diff
@@ -109,4 +121,40 @@
...
\ No newline at end of file
+}
+
+.volumeControlWrapper {
+  position: relative;
+  > i {
+    position: absolute;
+    z-index: 10;
+  }
+  > .volumeControlContainer {
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.scss]: `\n`

File: src/components/Core/Player/Player.scss [hunk 13]
```diff
@@ -109,4 +121,40 @@
...
+    z-index: 5;
+    background: #f7f7f7;
+    border-radius: 5px;
+    width: 30px;
+    height: 120px;
+    bottom: -5px;
+    box-shadow: 0 3px 10px -3px rgba(0, 0, 0, 0.25);
+    align-items: center;
+  }
+  &:hover > .volumeControlContainer {
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.scss]: `\n`

File: src/components/Core/Player/Player.scss [hunk 14]
```diff
@@ -109,4 +121,40 @@
       }
     }
   }
-}
\ No newline at end of file
+}
+
+.volumeControlWrapper {
+  position: relative;
+  > i {
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.scss]: `\n`

File: src/components/Core/Player/Player.scss [hunk 15]
```diff
@@ -56,13 +58,11 @@
...
     outline: none;
     padding: 0;
-    background:
-      #dedede
-      -webkit-gradient(
-        linear, 50% 0%, 50% 100%,
-        color-stop(0%, #fe2851),
-        color-stop(100%, #fe2851)
-      ) no-repeat;
+    background: #dedede -webkit-gradient(
```
Comment (raphaelvigee) [inline src/components/Core/Player/Player.scss]: Pls keep the old style

File: src/components/Core/Player/Player.jsx (hunk 4 above)
Comment (FilipGrebowski) [reply src/components/Core/Player/Player.jsx]: I use it for adding the classes tho.

**Pull request merged**